packages/react-scripts/template/README.md

```diff
@@ -1,6 +1,6 @@
 This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
 
-Below you will find some information on how to perform common tasks.  
```

Comment:
Whitespace is significant in Markdown.
This caused the linebreak to be ignored.
I'll fix this manually but please keep this in mind 😉